pom.xml

```diff
@@ -22,23 +22,24 @@
 
     <groupId>io.gravitee.policy</groupId>
     <artifactId>gravitee-policy-javascript</artifactId>
-    <version>1.2.0</version>
+    <version>1.2.1</version>
 
     <name>Gravitee.io APIM - Policy - Javascript</name>
     <description>Description of the Javascript Gravitee Policy</description>
 
     <parent>
         <groupId>io.gravitee</groupId>
         <artifactId>gravitee-parent</artifactId>
-        <version>20.0</version>
+        <version>21.0.0</version>
     </parent>
 
     <properties>
-        <gravitee-bom.version>1.0</gravitee-bom.version>
-        <gravitee-gateway-api.version>1.31.0</gravitee-gateway-api.version>
-        <gravitee-gateway-buffer.version>1.18.3</gravitee-gateway-buffer.version>
-        <gravitee-policy-api.version>1.9.0</gravitee-policy-api.version>
-        <gravitee-common.version>1.17.2</gravitee-common.version>
+        <gravitee-bom.version>3.0.8</gravitee-bom.version>
+        <gravitee-gateway-api.version>1.34.2</gravitee-gateway-api.version>
+        <gravitee-apim.version>3.18.28</gravitee-apim.version>
+        <gravitee-policy-api.version>1.11.0</gravitee-policy-api.version>
+        <gravitee-common.version>1.27.0</gravitee-common.version>
+        <gravitee-expression-language.version>1.9.7</gravitee-expression-language.version>
```

Comment:
Needed?

Reply:
yes, with the versions updates expression language is not provided anymore